fast64_internal/sm64/__init__.py

```diff
@@ -116,11 +125,63 @@ def draw(self, context):
 		prop_split(col, context.scene, 'convertibleAddr', 'Address')
 		col.prop(context.scene, 'levelConvert')
 
+def get_legacy_export_type():
+	legacy_export_types = ('C', 'Binary', 'Insertable Binary')
+	scene = bpy.context.scene
+
+	for exportKey in ['animExportType', 'colExportType', 'DLExportType', 'geoExportType']:
+		try:
+			eType = scene.pop(exportKey)
+			if eType and legacy_export_types[eType] != 'C':
+				return legacy_export_types[eType]
+		except KeyError:
+			pass
```

Comment:
```suggestion
		eType = scene.pop(exportKey, None)
		if eType is not None and legacy_export_types[eType] != 'C':
			return legacy_export_types[eType]
```
I didn't realize pop would raise an error if it doesn't get a default to return

about `eType is not None` see [PEP 8 Programming Recommendations](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)
> beware of writing `if x` when you really mean `if x is not None`

Reply:
ahh nice, changed!